orchestra/communication/staffing.py

```diff
@@ -206,6 +207,7 @@ def get_available_requests(worker):
         StaffBotRequest.objects
         .filter(inquiries__communication_preference__worker=worker)
         .exclude(status=StaffBotRequest.Status.CLOSED.value)
+        .exclude(task__project__status=Project.Status.COMPLETED)
```

Comment:
also exclude `Project.Status.ABORTED` (similar to Task status on the next two lines)